arrow-cast/src/cast.rs

```diff
@@ -704,15 +680,21 @@ fn as_time_res_with_timezone<T: ArrowPrimitiveType>(
 ///   in integer casts return null
 /// * Numeric to boolean: 0 returns `false`, any other value returns `true`
 /// * List to List: the underlying data type is cast
+/// * List to FixedSizeList: the underlying data type is cast. If safe is true and a list element
+/// has the wrong length it will be replaced with NULL, otherwise an error will be returned
```

Comment:
The documentation on `CastOptions` is pretty good

```suggestion
/// * List to FixedSizeList: the underlying data type is cast. If the list element
/// has the wrong length it will either be replaced with NULL, or an error will be returned depending on the
/// value of [`CastOptions::safe`]. 
```